[2024-04-25T07:57:24.425+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T07:57:24.443+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2024-04-25T07:57:24.449+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2024-04-25T07:57:24.450+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-25T07:57:24.462+0000] {taskinstance.py:2327} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-04-11 00:00:00+00:00
[2024-04-25T07:57:24.467+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3652) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T07:57:24.469+0000] {standard_task_runner.py:63} INFO - Started process 3656 to run task
[2024-04-25T07:57:24.468+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'insert_into_table', 'scheduled__2023-04-11T00:00:00+00:00', '--job-id', '4130', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpm1qjcaow']
[2024-04-25T07:57:24.470+0000] {standard_task_runner.py:91} INFO - Job 4130: Subtask insert_into_table
[2024-04-25T07:57:24.500+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2023-04-11T00:00:00+00:00 [running]> on host 4d994982806b
[2024-04-25T07:57:24.549+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-04-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-11T00:00:00+00:00'
[2024-04-25T07:57:24.550+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T07:57:24.558+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-04-11', 'dag_with_postgres_operator_v01')
        
[2024-04-25T07:57:24.562+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-25T07:57:24.568+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-25T07:57:24.572+0000] {sql.py:457} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-04-11', 'dag_with_postgres_operator_v01')
        , parameters: None
[2024-04-25T07:57:24.574+0000] {sql.py:466} INFO - Rows affected: 1
[2024-04-25T07:57:24.575+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-04-25T07:57:24.577+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T07:57:24.581+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=insert_into_table, execution_date=20230411T000000, start_date=20240425T075724, end_date=20240425T075724
[2024-04-25T07:57:24.610+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T07:57:24.618+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T07:57:24.619+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
